index.html

```diff
@@ -1644,6 +1634,170 @@ <h4>Background</h4>
+		methods of communication are not always available. When she’s offline and meets a new person 
+		she wishes to communicate with, she establishes a direct secure connection between her device 
+		and theirs by having them scan a QR code to pass on her DID and start sharing peer-to-peer 
+		messages secured by the keys on her device. When online, Samira’s DID can be resolved to its 
+		DID document that contains her public key material. In this way, she can maintain these secure 
+		connections whether she has access to the Internet or not. Whenever Samira is working on a new 
+		campaign or wants to raise awareness about issues affecting her local region, she is able to 
+		securely communicate with all relevant parties, regardless of the device, platform, or network 
+		infrastructure they are using. </p>
+<p>At one point, Samira sees a friend of hers, Ahmed, who’s part of a local organization campaigning against 
+	unfair labor practices in local agriculture. Samira wants to support the efforts of this group and get 
+	involved, but she worries that publicly allying with the farm workers will negatively affect her ability 
+	to act as a neutral party in labor negotiations. By creating a unique DID for herself in the group at very 
+	little cost, she can engage in online multi-party secure communication with all of the members of Ahmed’s 
+	organization. Messages are secured via mutual authentication between Samira and each of the other members. 
+	Using these connections, and using either direct peer to peer connections or online services that both Samir 
+	and Ahmed trust not to have any ‘backdoors’,  they are able to have private and confidential communications 
+	with the other members at any time, without fear of censorship. Members of the organization are able to have 
+	confidence that they’re talking to Samira, and she can verify who she is communicating with. Samira can use 
+	the same identifier and connection with the group to share credentials that prove she’s a humanitarian worker 
+	and attest to her history.</p>
+<p>Eventually, Samira loses her phone. To make sure no one impersonates her, she revokes the keys on all of the DIDs 
+	she is using for communication. Rather than losing access to all of her existing connections, Samira is able 
+	to update her online service endpoints and routing capabilities to make sure she continues to receive messages 
+	securely on the devices she still owns. When she gets a new phone, she can add new keys to her DID Document to 
+	continue exchanging secure messages online with her connections without interruption. Keys shared through direct, 
+	offline connections need to be updated one by one. Should Samira want to change the DIDs she’s using (for instance, 
+	because she’s changing DID method providers), she can send a message to her connections notifying them that she’s 
+	deprecating her old DID and providing her new DID with authentic digital signatures. Again, for offline 
```

Comment:
"Again, for offline communication, this must be done one by one but it means that Samira is able..."

Proposing to change to:
"Again, for entirely offline connections, this must be done one by one but it means that Samira is able..."